PKHeX/PKM/PK7.cs

```diff
@@ -544,7 +544,7 @@ public void FixMemories()
 
             if (GenNumber < 7) // must be transferred via bank, and must have memories
             {
-                TradeMemory(Bank: true);
+                TradeMemory(true);
```

Comment:
Oops, I actually didn't mean to push that, thanks for the catch. I meant to have that commit limited to casting changes, and I thought I took out everything I wasn't going to push, but I missed that.